### ISSUE TITLE
Expose Oauth handlers as Nitro plugin

### ISSUE THREAD

**Issue by adrienZ** [atinux/nuxt-auth-utils#167]
Today, handlers from src/runtime/server/lib/oauth are injected into the Nuxt context when the module is installed.

Would it be desirable to expand the scope of this package by providing those function directly in Nitro ? (i'm thinking [Nitro plugin](https://nitro.unjs.io/guide/plugins) or direct import).

This would help other communities (vinxi, solid, analog, etc) and library authors to easily extend this fantastic package :)

**Comment by Barbapapazes** [comment]
Yes. I think it's a good idea. I'm working of a bunch of Nitro modules for the v3 but I think that we still need to wait for the v3 and the kit package.

https://github.com/unjs/nitro/pull/2686#issuecomment-2305913286

**Comment by adrienZ** [comment]
Interesting, Nitro modules looks great indeed !

**Comment by atinux** [comment]
I think it would be even better to have `h3-oauth` exposing all our OAuth utils.
